.mailmap

```diff
@@ -122,8 +122,8 @@ Alan Bromborsky <abrombo@verizon.net> <brombo@GA.(none)>
 Alec Kalinin <alec.kalinin@gmail.com>
 Aleksandar Makelov <amakelov@college.harvard.edu>
 Alex Argunov <sajkoooo@gmail.com>
-Alex Lindsay <adlinds3@ncsu.edu> <lindsayad@users.noreply.github.com>
 Alex Lindsay <adlinds3@ncsu.edu> Alex Lindsay <alexlindsay239@gmail.com>
+Alex Lindsay <adlinds3@ncsu.edu> <lindsayad@users.noreply.github.com>
```

Comment:
I'm not sure why these kinds of changes sometimes happen. I guess you've used the `mailmap_update.py` script rather than the `mailmap_check.py` script but I can't be sure. Do you know which script you ran?

The authors check is failing and says this:
```
This author is not included in the .mailmap file:
Carson McManus <carson.mcmanus1@gmail.com>
This author is not included in the .mailmap file:
Carson McManus <dyc3@users.noreply.github.com>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.
The mailmap file was reordered

For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file
```
The lines that you should add to the .mailmap file would be something like:
```
Carson McManus <carson.mcmanus1@gmail.com> Carson McManus <dyc3@users.noreply.github.com>
```
That line means that `Carson McManus` with the gmail address is a listed author and that any commits with the `dyc3` email address should be associated with the same author.

After adding that line you should run the `mailmap_check.py` script twice: first time reorders the file and the second time should confirm that the file is now up to date.

Reply:
Ah yeah I did use the `mailmap_update.py` script. Seems like this process needs some better instructions haha